docs/embeddings/openai.md

```diff
@@ -45,14 +55,23 @@ openai_ef = embedding_functions.OpenAIEmbeddingFunction(
 
 ```javascript
 const {OpenAIEmbeddingFunction} = require('chromadb');
-const embedder = new OpenAIEmbeddingFunction({openai_api_key: "apiKey"})
+const embedder = new OpenAIEmbeddingFunction({
+    openai_api_key: "apiKey", 
+    model: "text-embedding-ada-002"
```

Comment:
do you think we should change the default to the new one?

Reply:
These are the pricing models.

Model | Usage
-- | --
text-embedding-3-small | $0.00002 / 1K tokens
text-embedding-3-large | $0.00013 / 1K tokens
ada v2 | $0.00010 / 1K tokens

And I think `small` outperforms `ada` and is cheaper, so it makes sense to change to SOTA, which is cheaper across the whole code base.

Reply:
@jeffchuber, I think this looks good now. I've considered upgrading the actual codebase to use the new models. However, that carries bigger risks as many people are not accustomed to providing model parameters to the EF, so the default behavior will break. On the other hand, we can warn the user (with their first use of the EF, at least for Python) to tell them there are newer, better, cheaper models.